public/components/model/model_details_panel.tsx

```diff
@@ -24,14 +24,10 @@ import { generatePath, useHistory } from 'react-router-dom';
 import { useOpenSearchDashboards } from '../../../../../src/plugins/opensearch_dashboards_react/public';
 import { mountReactNode } from '../../../../../src/core/public/utils';
 import { routerPaths } from '../../../common';
-import {
-  ErrorCallOut,
-  MODEL_NAME_FIELD_DUPLICATE_NAME_ERROR,
-  ModelDescriptionField,
-  ModelNameField,
-} from '../common';
+import { ErrorCallOut, ModelDescriptionField, ModelNameField } from '../common';
 
 import { BottomFormActionBar } from './bottom_form_action_bar';
+import { MODEL_NAME_FIELD_DUPLICATE_NAME_ERROR } from '../common/forms/form_constants';
```

Comment:
Could we move this import before `./`? That's my person import order
```tsx
// import from "node_modules"
// import from "parent relative paths"
// import from "current directory"

```

Reply:
Sounds good, we can even have editor import sort plugin for it. I will update it manually for now.